phidgets_spatial/src/spatial_ros_i.cpp

```diff
@@ -0,0 +1,295 @@
+#include <functional>
```

Comment:
We should probably add a "migration guide" for people upgrading to Noetic. Here are the entries for the Phidgets Spatial:

- parameters renamed:

    * `period -> data_interval_ms`
    * `serial_number -> serial`

- frame_id: default changed "imu" -> "imu_link" as specified in http://www.ros.org/reps/rep-0145.html
- diagnostics are gone (but will hopefully be added back later)